src/style/app.module.css

```diff
@@ -1942,185 +2058,794 @@ input[type='radio']:checked + label:hover {
   border-radius: 5px;
-  background-color: var(--bs-primary);
-  width: 100%;
-  font-size: 16px;
-  color: var(--bs-white);
-  outline: none;
-  font-weight: 600;
-  cursor: pointer;
-  transition:
-    transform 0.2s,
-    box-shadow 0.2s;
-  width: 100%;
 }
 
-.manageBtn {
-  margin: 1rem 0 0;
-  margin-top: 15px;
-  border: 1px solid var(--grey-border-box-color);
-  box-shadow: 0 2px 2px var(--grey-border-box-color);
-  padding: 10px 10px;
-  border-radius: 5px;
-  font-size: 16px;
-  color: var(--bs-white);
-  outline: none;
-  font-weight: 600;
-  cursor: pointer;
-  width: 45%;
-  transition:
-    transform 0.2s,
-    box-shadow 0.2s;
+.borderRounded8 {
+  border-radius: 8px;
 }
 
-.removeFilterIcon {
-  cursor: pointer;
+.topRadius {
+  border-top-left-radius: 24px;
+  border-top-right-radius: 24px;
 }
 
-.searchForm {
-  display: inline;
+.bottomRadius {
+  border-bottom-left-radius: 24px;
+  border-bottom-right-radius: 24px;
 }
 
-.view {
-  margin-left: 2%;
-  font-weight: 600;
-  font-size: 16px;
-  color: var(--bs-gray-600);
+.shadow {
+  box-shadow: 5px 5px 4px 0px #31bb6b1f;
 }
 
-/* header (search, filter, dropdown) */
-.btncon .btnsContainer {
-  display: flex;
-  margin: 0.5rem 0 1.5rem 0;
+.mainContainerUserPeople {
+  width: 50%;
+  flex-grow: 3;
 }
 
-.btncon .btnsContainer .input {
-  flex: 1;
-  min-width: 18rem;
-  position: relative;
+.semiBold {
+  font-weight: 500;
 }
 
-.btncon .btnsContainer input {
-  outline: 1px solid var(--bs-gray-400);
+.placeholderColor::placeholder {
+  color: #737373;
 }
 
-.btncon .btnsContainer .input button {
-  width: 52px;
+.personDetails {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
 }
 
-.noOutline input {
-  outline: none;
+.personImage {
+  border-radius: 50%;
+  margin-right: 20px;
+  max-width: 70px;
 }
 
-.noOutline input:disabled {
-  -webkit-text-fill-color: var(--black-color) !important;
+.blueText {
+  color: var(--light-blue);
 }
 
-.noOutline textarea:disabled {
-  color: var(--black-color) !important;
-  opacity: 1;
+.accordionSummary {
+  width: 100% !important;
+  padding-right: 0.75rem;
+  display: flex;
+  justify-content: space-between !important;
+  align-items: center;
 }
 
-.inputFields {
-  box-shadow: 0 1px 1px var(--brand-primary);
+.accordionSummary button {
+  height: 2.25rem;
+  padding-top: 0.35rem;
 }
 
-.dropdowns {
-  background-color: var(--bs-white);
-  border: 1px solid var(--dropdown-border-color);
-  position: relative;
-  display: inline-block;
-  color: black;
+.accordionSummary button:hover {
+  /* background-color: #31bb6a50 !important; */
+  /* color: #31bb6b !important; */
 }
 
-.chipIcon {
-  height: 0.9rem !important;
+.subContainer span {
+  font-size: 0.9rem;
+  margin-left: 0.5rem;
+  font-weight: lighter;
+  color: #707070;
 }
 
-.chip {
-  height: 1.5rem !important;
+.box {
+  width: auto;
+  /* height: 200px; */
+  background-color: white;
+  margin-top: 1rem;
+  padding: 20px;
+  border: 1px solid #dddddd;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`